CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
-## [next-version]
+## [6.15.5] - 2021-12-2
```

Comment:
The `[next-version]` header is missing

Reply:
Just the release date should be updated

Reply:
This release went out on Thurs, 2 Dec 2021 though. So the date here is correct. 🤔